docker/Dockerfile.starwhale

```diff
@@ -4,6 +4,13 @@ FROM ${BASE_IMAGE}
 ARG SW_VERSION=0.1.0
 ENV SW_VERSION ${SW_VERSION}
 
+ARG NEXUS_HOSTNAME=host.nexus
```

Comment:
in [sw-docker-entrypoint](https://github.com/star-whale/starwhale/blob/main/client/scripts/sw-docker-entrypoint#L28), there are some pypi config environment fields.  When running a container,  you can set `SW_PYPI_INDEX_URL`, `SW_PYPI_EXTRA_INDEX_URL` and `SW_PYPI_TRUSTED_HOST` envs.

Reply:
line 7 ~ line11 omitted?

Reply:
when build docker image, this pip.conf is used too.

Reply:
ok.